packages/blade/src/components/Input/BaseInput/BaseInput.tsx

```diff
@@ -423,7 +423,7 @@ export const BaseInput = React.forwardRef<HTMLInputElement, BaseInputProps>(
             <BaseInputVisuals leadingIcon={leadingIcon} prefix={prefix} isDisabled={isDisabled} />
             <StyledBaseInput
               id={inputId}
-              ref={ref}
+              ref={ref as never}
```

Comment:
Didn't understand why do we need `as never` here?

Reply:
meant to do `as any`